src/pfs/pfs_test.go

```diff
@@ -0,0 +1,16 @@
+package pfs
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestProject_ValidateName(t *testing.T) {
```

Comment:
Is it idiomatic to have underscores in test names?

Reply:
I stole the pattern from [example names](https://pkg.go.dev/testing#hdr-Examples), which for methods are of the form `ExampleProject_ValidateName`. Granted, it is really weird to see an underscore in a name in Go.